Neos.Fusion/Classes/FusionObjects/AbstractArrayFusionObject.php

```diff
@@ -39,7 +39,7 @@ abstract class AbstractArrayFusionObject extends AbstractFusionObject implements
      *
      * @var array
      */
-    protected $ignoreProperties = [];
+    protected $ignoreProperties = ['__meta'];
```

Comment:
so including `__meta` here is now crucial? because of the new ignore filter logic?

havent checked it, but when the runtime calls `setIgnoreProperties()` will it also add the `__meta` value?

Reply:
Ah, damn forgot to remove that, I think it's not necessary, but we should talk about ignore and __meta because IMHO it behaves weirdly